drivers/clock_control/Kconfig.stm32

```diff
@@ -106,7 +106,7 @@ config CLOCK_STM32_PLL_PREDIV
 
 config CLOCK_STM32_PLL_PREDIV1
 	int "PREDIV1 Prescaler"
-	depends on CLOCK_STM32_PLL_SRC_HSE && (SOC_STM32F091XB || SOC_STM32F091XC)
+	depends on CLOCK_STM32_PLL_SRC_HSE && (SOC_STM32F091XB || SOC_STM32F091XC || SOC_STM32F072XB)
```

Comment:
Isn't that valid for all F0X ?

Reply:
Will have a look

Reply:
Seems only needed by the following parts:
stm32f030xc.h
stm32f042x6.h
stm32f048xx.h
stm32f070x6.h
stm32f070xb.h
stm32f071xb.h
stm32f072xb.h
stm32f078xx.h
stm32f091xc.h
stm32f098xx.h

And is not needed by:
stm32f030x6.h
stm32f030x8.h
stm32f031x6.h
stm32f038xx.h
stm32f051x8.h
stm32f058xx.h

Reply:
Then I think we should define a sub series or a CONFIG(CLOCK_STM32_PLL_PREDIV1_SUPPORT?) option to deal with this. Such as you've done for I2C.
(Unless you can find a clean way to get info from soc/stm32f0....h file)

Reply:
All the socs supporting PREDIV1 declares RCC_PLLSRC_PREDIV1_SUPPORT, and it's only when the SoC has an HSE.
So we can say we remove the && (SOC_STM32F091XB || ...) and consider the config will be correct.
If someone tries to enable HSE on a invalid SoC, it won't harm because the code won't use the PREDIV1 value and the SoC will take the HSI source be default.

Reply:
Right, will work. 
To avoid later confusion, could you document it  config help section?

Reply:
Do you mean add a help block under CLOCK_STM32_PLL_PREDIV1 config ?

Reply:
yes

Reply:
Ok done